contracts/package.json

```diff
@@ -38,6 +38,7 @@
   "author": "Origin Protocol Inc <support@originprotocol.com>",
   "license": "MIT",
   "devDependencies": {
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.9",
```

Comment:
nice addition. We can start following Hardhat's recommendation and migrate away from Waffle with this plug-in https://hardhat.org/hardhat-runner/docs/guides/migrating-from-hardhat-waffle

Reply:
Have tried quickly doing it by following the migration guide but got an error: 
![Screenshot 2023-10-24 at 16 45 10](https://github.com/OriginProtocol/origin-dollar/assets/579910/5e7730a0-c815-4567-96e8-7d0b8b5878f5)

have created a [Notion task](https://www.notion.so/originprotocol/Migrate-from-Waffle-to-hardhat-27c543b54a91421b8bbd2e2f1f4be617
) for it: